flake8_quotes/__init__.py

```diff
@@ -44,19 +44,27 @@ def __init__(self, tree, filename='(none)', builtins=None):
     def add_options(cls, parser):
         parser.add_option('--quotes', default='\'', action='store',
                           help='Quote to expect in all files (default: \')')
-        parser.add_option('--multiline-quotes', default='\'', action='store',
-                          help='Multiline quote to expect in all files (default: \')')
+        parser.add_option('--multiline-quotes', default='', action='store',
+                          help='Multiline quote to expect in all files (default: any)')
         parser.config_options.extend(['quotes', 'multiline_quotes'])
 
     @classmethod
     def parse_options(cls, options):
+        multiline_ignored = True
+
         if hasattr(options, 'multiline_quotes'):
             multiline_quotes = options.multiline_quotes
+
+            if len(multiline_quotes) == 0:
+                multiline_quotes = options.quotes
+            else:
+                multiline_ignored = False
         else:
             multiline_quotes = options.quotes
 
-        cls.quotes = cls.QUOTES[options.quotes]
+        cls.quotes = cls.INLINE_QUOTES[options.quotes]
```

Comment:
For consistency, this should be `cls.inline_quotes` as well as our other references to anything like quotes (e.g. `tests`, CLI option). However, maybe that will be too noisy for this PR. Thoughts?